### PR TITLE
Changed the format of clan war message

### PR DESCRIPTION
**Pull request by sivykh**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -44,17 +44,22 @@ def load_clan_war_info(clan_tag):
 
     sorted_players = []
     for tag, participants in clan_war_info.items():
-        wins = 0
-        battles = 0
+        wins = 0.0
```
Comment (dfomin) [inline main.py:47]: why?

File: main.py [hunk 2]
```diff
@@ -44,17 +44,22 @@ def load_clan_war_info(clan_tag):
...
-
             if participant is not None:
                 wins += participant['wins']
                 battles += participant['battlesPlayed']
-                plays += "1" if participant['wins'] > 0 else "0" if participant['battlesPlayed'] > 0 else "_"
+                if participant['battlesPlayed'] < 2:
+                    plays += "1" if participant['wins'] > 0 else "0" if participant['battlesPlayed'] > 0 else "_"
+                else:
+                    plays += ")"
+                    for warBattle in range(participant['battlesPlayed']):
```
Comment (dfomin) [inline main.py]: use python code style - war_battle

**Review by dfomin — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (sivykh) [reply main.py:47]: Because it declares integer number type instead of real number type. It doesn't work on python 2.7 and earlier

Comment (dfomin) [reply main.py:47]: Python 2 and 3 is not compatible, we will not support both versions.

**Pull request merged**